src/server.tsx

```diff
@@ -370,7 +368,7 @@ app.get(
 
 // TODO remove once migration complete
 app.post('/epic/compare-variant-decision', async (req: express.Request, res: express.Response) => {
-    if (process.env.LOG_COMPARE_VARIANTS !== 'true') {
```

Comment:
Not for this PR, but this reminds me we're probably at the point where we could lose this functionality.